.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,55 @@
+name: CI/CD
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+
+      - name: NPM install
+        run: npm run clean:install
+        working-directory: source/constructs
+      - name: Build image handler
+        run: npm run build
+        working-directory: source/image-handler
+
+      # Deploy cloudformation stack with CDK
+      - name: Set QA Environment
+        if: github.base_ref == 'qa'
+        run: |
+          echo "STAGE=qa" >> $GITHUB_ENV
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.LOWER_ENV_AWS_ACCESS_KEY_ID }}" >> $GITHUB_ENV
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.LOWER_ENV_AWS_SECRET_ACCESS_KEY }}" >> $GITHUB_ENV
+          echo "SOURCE_BUCKET=mslc-player-qa-document-upload" >> $GITHUB_ENV
+          echo "S3_KMS_KEY_ARN=arn:aws:kms:us-east-1:963973445526:key/9ea6ef14-c12a-42c6-9e6d-56cc0f389bb4" >> $GITHUB_ENV
+          echo "SIGNING_SECRET=mslc-document-upload-qa" >> $GITHUB_ENV
+      - name: Set UAT Environment
+        if: github.base_ref == 'uat'
+        run: |
+          echo "STAGE=uat" >> $GITHUB_ENV
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.LOWER_ENV_AWS_ACCESS_KEY_ID }}" >> $GITHUB_ENV
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.LOWER_ENV_AWS_SECRET_ACCESS_KEY }}" >> $GITHUB_ENV
+          echo "SOURCE_BUCKET=mslc-player-uat-document-upload" >> $GITHUB_ENV
+          echo "S3_KMS_KEY_ARN=arn:aws:kms:us-east-1:963973445526:key/60c48e18-9e6c-4f74-84c7-5241fd45eca3" >> $GITHUB_ENV
+          echo "SIGNING_SECRET=mslc-document-upload-uat" >> $GITHUB_ENV
+      - name: Set Prod Environment
+        if: github.base_ref == 'master'
+        run: |
+          echo "STAGE=prod" >> $GITHUB_ENV
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.PROD_AWS_ACCESS_KEY_ID }}" >> $GITHUB_ENV
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}" >> $GITHUB_ENV
```

Comment:
do we need `S3_KMS_KEY_ARN` and `SOURCE_BUCKET` here?

Reply:
Yes, I need to fill in the whole production config, need some help (probably from alex) to set that up